.github/workflows/conformance-multicluster.yaml

```diff
@@ -322,6 +322,8 @@ jobs:
 
       - name: Wait for cluster mesh status to be ready
         run: |
+          cilium status --wait --context ${{ steps.contexts.outputs.context1 }}
+          cilium status --wait --context ${{ steps.contexts.outputs.context2 }}
```

Comment:
While this is fine for now, I suppose a more proper fix would be to have the `clustermesh status --wait` command automatically also imply a `status --wait` directly in CLI code. cc @tklauser for thoughts.

Reply:
Well, ultimately, they do different things. Ideally there would be a `--wait` in `clustermesh join`